scripts/test_cases/large_app_with_private_packages_centralised_ngbv_removed_linux.py

```diff
@@ -0,0 +1,28 @@
+"""_summary_
+This script basically runs benchmark test for msbuild
+Returns:
+    None: .....
+"""
+
+import subprocess
+
+EXTRACT_PATH = "sdk"
+WORKING_DIR = "msbuild-performance-test"
+DOTNET_BASE_VERSION_URL_LINUX = "https://download.visualstudio.microsoft.com/download/pr/dc930bff-ef3d-4f6f-8799-6eb60390f5b4/1efee2a8ea0180c94aff8f15eb3af981/dotnet-sdk-6.0.300-linux-x64.tar.gz"
+DOTNET_DAILY_VERSION_URL_LINUX = "https://aka.ms/dotnet/8.0.1xx/daily/dotnet-sdk-linux-x64.tar.gz"
+TEST_SOLUTION_REPO_URL = "https://github.com/marcin-krystianc/TestSolutions.git"
```

Comment:
See NuGet benchmarks, they check out the repo at specific commit hash (e.g.: https://github.com/G-Research/DotNetPerfMonitor/blob/main/scripts/perftests/testCases/LargeAppCPM64-142722b.ps1#L5).
It is important to do that because we want to measure performance for exactly the same state of the repo each time.